main.go

```diff
@@ -170,6 +180,17 @@ func run(cmd *cobra.Command, _ []string) {
 		managerOptions.Namespace = watchNamespace
 	}
 
+	if observabilityServiceEndpoint != "" {
+		otel.SetLogger(logger)
+		tp, err := trace.CreateTraceProvider(observabilityServiceEndpoint, version, observabilityServiceSeed)
+		if err != nil {
+			logger.Error(err, "unable to create traceprovider")
+			os.Exit(1)
+		}
+		otel.SetTracerProvider(tp)
+		otel.SetTextMapPropagator(otel_propagation.NewCompositeTextMapPropagator(otel_propagation.TraceContext{}, otel_propagation.Baggage{}))
```

Comment:
Worth enabling propagation even when observability reporting is off? Right now, it's not injecting trace IDs or baggage into outgoing requests (e.g. HTTP requests to external sources of metadata), but it'll be useful once added.